x-pack/solutions/security/plugins/security_solution/public/entity_analytics/pages/entity_store_management_page.tsx

```diff
@@ -266,7 +248,7 @@ export const EntityStoreManagementPage = () => {
                 </p>
               </EuiCallOut>
             )}
-            {callouts}
+            {selectedTabId === TabId.Import && callouts}
```

Comment:
This is why they were so squashed before, this is designed for a two panel style UI but the status page doesn't have room.